### DISCUSSION TITLE
M.I.B. Installation Issues

### DISCUSSION THREAD

**Discussion by staryknight8** [Mr-MIBonk/M.I.B._More-Incredible-Bash#77] [Q&A]
Hi guys, new to the forum and am trying to get M.I.B working for my Audi C7 which is currently on Firmware MHI2_ER_AU57x_K3663_1 which I paid someone to install previously to enable carplay/android auto. 

I have recently moved countries from the UK to Australia, and the car is currently set to Europe region, and have been trying to change the region to ROW to enable the ROW maps to install. I tried originally with another MIB2 software called MIB2 Toolbox, and after trying that and it didnt work, i removed it from my MMI to try and install M.I.B. I have tried four times to install M.I.B, 3 manual and 1 auto, and each time it goes through all the right steps and confirms the installation of the "Launcher" was "OK", but everytime i go into the green menu the M.I.B menu options are not there at all. 

I'm looking for some advice as to why the M.I.B wont install, did i stuff it up after installing the MIB2 Toolbox and removing it? any advice is much appreciated, thanks.

**Comment by harman-f** [comment]
Having something done to your unit before is always bad.

Especially, as the MHI2_ER_AU57x_K3663_1 is a free solution!
Best would be to get the latest official MHI2_ER_AU57x_K3663_1 version and install it over yours.
https://mibwiki.one/doc/mhi2_er_au57x_k3663_1-mu1425-aio-R6NuQH8h5p

Install latest M.I.B after that, create a backup and share it with me.

We go from there ;)

**Reply by staryknight8** [reply]
ok thanks will do!

**Reply by staryknight8** [reply]
Edit: I was just an idiot and kept trying to access the GEM without the M.I.B SD card inserted so it never showed up...

On a side note, the AIO firmware update did install the FEC "023d00ee" into my activation keys list? now im trying to troubleshoot why my system still wont let me update the ROW maps, keeps saying the update data is invalid or something..

**Reply by harman-f** [reply]
you also have to adjust coding.
